### PR TITLE
Brings us up to 512.

### PR DESCRIPTION
**Pull request by FlattestGuitar**
* Did you update Travis to compile for 512?
   - No, because 512 is not a stable release yet.

* Does this mean I can now code and compile Oracle with 512?
   - Yes. No need to use 511 anymore.

* Does this PR actually introduce any meaningful changes?
   - Yes. I took the ooportunity to port /tg/'s gas transparency, so gas no longer blocks you clicking tiles underneath it. We can switch to a different set of gas sprites once 512 is stable.

* Does this mean I can now play the game with 512?
   - You always could.

### CODE REVIEW

**Comment by Ziiro** [comment]
WIDESCREEN WHEN

**Comment by SamHPurp** [comment]
Please reference the PRs you are porting - as this then pops up on their Git and helps me know which ones I don't need to bother porting.

**Comment by SamHPurp** [comment]
@FlattestGuitar please link to the PRs you have ported, as per previous request/reasoning.

**Comment by FlattestGuitar** [comment]
https://github.com/tgstation/tgstation/pull/31251 This is for the first two commits

https://github.com/tgstation/tgstation/pull/32569 This is gas transparency

**Comment by SamHPurp** [comment]
iirc I heard things about gas stuff breaking and them having to revert / disable. Dunno if connected.

File: code/game/machinery/computer/apc_control.dm [hunk 1]
```diff
@@ -51,19 +51,19 @@
 	if(authenticated)
 		if(!checking_logs)
 			dat += "Logged in as [auth_id].<br><br>"
-			dat += "<i>Filters</i><br>"
-			dat += "<b>Name:</b> <a href='?src=[REF(src)];name_filter=1'>[filters["Name"] ? filters["Name"] : "None set"]</a><br>"
-			dat += "<b>Charge:</b> <a href='?src=[REF(src)];above_filter=1'>\>[filters["Charge Above"] ? filters["Charge Above"] : "NaN"]%</a> and <a href='?src=[REF(src)];below_filter=1'>\<[filters["Charge Below"] ? filters["Charge Below"] : "NaN"]%</a><br>"
-			dat += "<b>Accessible:</b> <a href='?src=[REF(src)];access_filter=1'>[filters["Responsive"] ? "Non-Responsive Only" : "All"]</a><br><br>"
+			dat += "<i>result_filters</i><br>"
```
Comment (ike709) [inline code/game/machinery/computer/apc_control.dm]: This should remain as `Filters`, actually.

**Comment by SamHPurp** [comment]
Lets make a mistake!

**Pull request merged**